project/Build.scala

```diff
@@ -82,7 +82,7 @@ object Build extends sbt.Build {
     organization := "dcos",
     scalaVersion := "2.11.8",
     crossScalaVersions := Seq(scalaVersion.value),
-    scalacOptions in Compile ++= Seq(
+    scalacOptions in (Compile, doc) ++= Seq(
```

Comment:
what is the doc here?

Reply:
this is the configuration necessary to added the option:  `"-no-link-warnings" // Suppresses problems with Scaladoc @throws links`